tensorboard/webapp/core/store/core_reducers.ts

```diff
@@ -112,6 +115,6 @@ const reducer = createReducer(
   })
 );
 
-export function reducers(state: CoreState, action: Action) {
+export function reducers(state: CoreState | undefined, action: Action) {
```

Comment:
To help me understand: why is this change necessary?

Reply:
Correct. When injected the state via third arguments in the `StoreModule.forFeature`, it looks like the `state` can be undefined.